common/djangoapps/student/models.py

```diff
@@ -763,7 +763,7 @@ class Meta(object):
         db_table = "auth_registration"
 
     user = models.OneToOneField(User, on_delete=models.CASCADE)
-    activation_key = models.CharField(('activation key'), max_length=32, unique=True, db_index=True)
+    activation_key = models.CharField(_(u'activation key'), max_length=32, unique=True, db_index=True)
```

Comment:
Why are you adding the underscores here?  We don't want to internationalize the names of model fields.